source/jobs/ra-2016-imaging-cvt.rst

```diff
@@ -0,0 +1,53 @@
+Research Assistant Opportunity (2016): Synthetic Image Generation for Nuclear Non-proliferation
+==================================================================================================
+
+*[Updated: March 31, 2016]*
+
+The Computational Nuclear Engineering Research Group (CNERG) at UW Madison is
+currently working at the interface of international security policy and nuclear
+engineering on issues related to treaty verification for fissile material
+production, such as the Non-Proliferation Treaty or a potential Fissile Material
+Cutoff Treaty.  We have developed the `Cyclus fuel cycle simulator
+<http://fuelcycle.org>`_  to model current and next-generation nuclear fuel
+cycles as part of the `Consortium for Verification Technology
+<http://cvt.engin.umich.edu/>`_. |Cyclus| is an agent-based, systems-level
+simulator that tracks discrete material flow through the entire fuel cycle,
+from mining through burnup in reactors to a repository.  
+
+We seek a research assistant to develop synthetic satellite images of nuclear
+facilities for monitoring and verification.  Satellite images can provide
+information on material shipments, production and maintenance staffing, physical
+ and environmental impact, etc.  Synthetic images will be designed with
+ changeable features at different times that are correlated with other fuel
+ cycle signals.  These images will be used in Cyclus to study the clandestine
+ material diversion from declared fuel cycles.  Due to funding restrictions,
+ only US citizens can be considered.
+
+Experience
+----------
+
+* Masters or PhD student in Nuclear Engineering, Computer Science, Physics,
+  Math or related field
+* Interest in international security and nuclear non-proliferation OR
+* Experience with image processing and analysis
+* No programming experience required, but must be willing to learn 
+
+
+Specific Duties
+---------------
+
+* Research appropriate parameters and specifications to make simplified but
+  representative fake satellite images (resolution, image size, variety of
+  information)
+* Define the properties of relevant image features (such as truck shape/size,
+  etc)
+* Write a program in Python that uses a table of features and positions to
+  create fake images. Program will be flexible/extensible for additional
+  features in the future.
+
+To be considered for an hourly Research Assistant position (50% plus tuition
+remission), please send a letter of interest, resume, and unofficial
+transcript to wilsonp _at_ engr.wisc.edu.  For further questions please
```

Comment:
Maybe we can use the address: cnerg-cvt@lists.wisc.edu   
I'm pretty sure it works...